packages/blade/src/components/Icons/Icons.stories.tsx

```diff
@@ -43,6 +43,51 @@ const Page = (): ReactElement => {
         export default App;
         `}
       </Sandbox>
+      <Title>Adding icons</Title>
+      <Description>
+        Please validate in the all icons story below in case the icon you wish to add is already
+        added. Steps for adding a new icon from Figma:
+      </Description>
+      <ul>
+        <li>
+          <Description>
+            Visit the [icons figma
+            file](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=57%3A0&t=xzz6HfFC50U8iuJp-0)
+          </Description>
+        </li>
+        <li>
+          <Description>
+            Click the icon you wish to add on Figma. Select the root icon element and not the layer
+            within. Copy this root icon as SVG. **Note**: generally the root svgs have a viewbox of
+            `0 0 24 24`.
+          </Description>
+        </li>
+        <li>
+          <Description>
+            ![](https://user-images.githubusercontent.com/24487274/203279145-1e0b0540-467d-4901-97c8-014f98b3cfca.png)
+          </Description>
+        </li>
+        <li>
+          <Description>
+            Replace the native HTML elements with Blade's components (eg. `svg` becomes `Svg`). You
+            may use a tool such as [SVGR](https://react-svgr.com/playground/?native=true) for the
+            initial transformation. Some properties may still need to be adjusted. Follow an
+            existing icon as a reference.
+          </Description>
+        </li>
+        <li>
+          <Description>
+            Once you're done making changes, run the storybook to verify how the new icon is looking
```

Comment:
```suggestion
            Once you're done making changes, run the storybook to verify how the new icon looks
```

Reply:
Done in https://github.com/razorpay/blade/pull/883/commits/6bd812011991d0077ccfdbc28db1625c68607145